### ISSUE TITLE
I can't reproduce the results of tf-xlm-r-ner-40-lang model 

### ISSUE THREAD

**Issue by isl-m** [huggingface/transformers#6615]
I tired to reproduce the results of tf-xlm-r-ner-40-lang model but there were compatibility issues in the token-classification/run_tf_ner.py
I fixed some of these issue but it's still not running as expected, @jplu could you please share with me the transformers/tf version used to produce the results of tf-xlm-r-ner-40-lang model.

i'm using:
transformers = 3.0.2
tensorflow = 2.3.0

Thanks in advance.

**Comment by jplu** [comment]
Hello!

What is the command line you use to train the model?

**Comment by isl-m** [comment]
cd examples/ner
python run_tf_ner.py \
--data_dir . \
--labels ./labels.txt \
--model_name_or_path jplu/tf-xlm-roberta-base \
--output_dir model \
--max-seq-length 128 \
--num_train_epochs 2 \
--per_gpu_train_batch_size 16 \
--per_gpu_eval_batch_size 32 \
--do_train \
--do_eval \
--logging_dir logs \
--mode token-classification \
--evaluate_during_training \
--optimizer_name adamw

**Comment by jplu** [comment]
Can you use the last version of the trainer, with th ecommand line I used:

```
python run_tf_ner.py \
--data_dir . \
--labels ./labels.txt \
--model_name_or_path jplu/tf-xlm-roberta-base \
--output_dir model \
--num_train_epochs 8 \
--per_gpu_train_batch_size 32 \
--per_gpu_eval_batch_size 64 \
--do_train \
--do_eval \
--do_predict \
--logging_steps 10 \
--evaluate_during_training \
--save_steps 100 \
--overwrite_output_dir
```
